src/components/atoms/CommentProfile/CommentProfile.style.ts

```diff
@@ -1,15 +1,31 @@
 import { css } from '@emotion/react';
 import color from '@/styles/color';
 
-export const containerStyle = (stance: 'A' | 'B') => css`
+export const containerStyle = css`
   display: inline-flex;
   padding: 5px;
   align-items: center;
   gap: 10px;
   border-radius: 50px 50px 0 50px;
-  background: ${stance === 'A' ? color.RED : color.BLUE};
 `;
 
+export const getProfileColor = (option: 'A' | 'B' | null) => {
+  if (option === 'A') {
+    return css({
+      backgroundColor: color.RED,
+    });
+  }
+  if (option === 'B') {
+    return css({
+      backgroundColor: color.BLUE,
+    });
+  }
+
+  return css({
+    backgroundColor: color.GY[2],
+  });
+};
+
```

Comment:
조건문을 최소화하고 객체 매핑을 사용해서 스타일을 작성하는건 어떨까용? 

```suggestion
export const getProfileColor = (option: 'A' | 'B' | null) => {
  const backgroundColors = {
     A: color.RED,
     B: color.BLUE,
     null: color.GY[2],
  };
  
  return css({
     backgroundColor: backgrounColors[option ?? 'null'],
  });
};
```

Reply:
[f871720](https://github.com/CHZZK-Study/Balance-Talk-Frontend/pull/202/commits/f871720760213354114adff7d92b9c48ff489251) 에서 스타일 수정했습니다!! 훨씬 간결해보이네용 👏

Reply:
오오 이 방법 저도  참고해야겠네요 너무 깔꼼쓰 👍